source/VMelnalksnis.PaperlessDotNet.DependencyInjection/ServiceCollectionExtensions.cs

```diff
@@ -83,7 +83,8 @@ static ServiceCollectionExtensions()
 				var httpClient = provider.GetRequiredService<IHttpClientFactory>().CreateClient(PaperlessOptions.Name);
 				var options = provider.GetRequiredService<PaperlessJsonSerializerOptions>();
 				var taskClient = provider.GetRequiredService<ITaskClient>();
-				return new(httpClient, options, taskClient);
+				var paperlessOptions = provider.GetRequiredService<IOptions<PaperlessOptions>>();
```

Comment:
Since the client is registered as scoped, we should respect configuration changes. `IOptions` is read only at startup, and `IOptionsSnapshot` reads from `IConfiguration` for each scope, instead of only on change, so I think `IOptionsMonitor` is the best choice here.
```suggestion
				var paperlessOptions = provider.GetRequiredService<IOptionsMonitor<PaperlessOptions>>().CurrentValue;
```

Reply:
what do you think about a property in the `DocumentClient` that can be set by anyone but is also configured by the `IConfiguration` when unsing the DI extensions?

Reply:
I'd rather have a single way of configuring the client using `PaperlessOptions`.